initial-configuration/jenkins/jenkins-docker/Dockerfile

```diff
@@ -26,4 +26,12 @@ RUN apt-get install jq -y
 
 RUN apt-get -y install uuid-runtime
 
+#Installing chrome for running selenium test
```

Comment:
Make this configurable so that there is a jenkins-test and jenkins images, build:
- Pull out env vars into a directory that is not in version control so they don't get trampled when we pull the aio
- Add a new env var for jenkins/jenkins-test differntiation 
- When doing the initial config, build the correct image based off that var
- In start-jenkins.sh, run the correct image based off that var
- In update-jenkins.sh build the correct image based off that var